### ISSUE TITLE
install.bat not running on windows 11

### ISSUE THREAD

**Issue by mrsaadfazal** [rupeshs/fastsdcpu#185]
Getting this error when i try to run the install.bat
`
Starting FastSD CPU env installation...
Python command check :OK
Python version: 3.11.7
The system cannot find the path specified.
The system cannot find the path specified.
FastSD CPU env installation completed.
Press any key to continue . . .
`

**Comment by mrsaadfazal** [comment]
I resolved the error by uninstalling Python from the Windows Store and opting for a version directly from the official website. The problem stemmed from my system using a Python interpreter from a MinGW installation located within `msys64`. This setup is more Unix-like and was conflicting with how I expected Windows to handle Python environments.

Originally, the Python path was set to:

`C:\msys64\mingw64\bin`

Because of this, when I created a virtual environment with `python -m venv env`, the directory structure looked like this:

- `env/lib`
- `env/bin` (on Windows, I expected this to be `Scripts`, but `bin` is typical for Linux and Mac)
- `env/include`

After uninstalling this version of Python and installing it from the official website, the issue was resolved. Now, Python is installed at:

`C:\Users\userName\AppData\Local\Programs\Python\Python311\python.exe`

Everything works as expected now, with the proper directory structure for Python virtual environments on Windows.
